src/js/main.js

```diff
@@ -421,37 +426,82 @@ Site.Map = {
     var newX = _this.mapPosition[4] + (translation * Math.cos(angle));
     var newY = _this.mapPosition[5] + (translation * Math.sin(angle));
 
+    // Move the map
+    _this.moveMap(newX,newY);
+
+    // Animate recursevly
+    window.requestAnimationFrame(_this.pan.bind(_this));
+
+  },
+
+  // Check if golocation is available and move the map to that equivalent location
+  geolocation: function() {
+    var _this =  this;
+
+    if (!WP.clientGeolocation) {
+      console.log('Geolocation not returned by the server');
+    } else if (WP.clientGeolocation.latitude && WP.clientGeolocation.longitude) {
+
+      // NOTES:
+      //
+      // # LONGITUDE
+      //
+      // Goes -180 to 180 deg. 0 is Greenwich. Negative is to the east,
+      // positive is west. In the map context it goes from 0 to
+      // _this.window.width * -2.
+      // Equivalent to the X axis
+      //
+      // # LATITUDE
+      //
+      // Goes from -90 to 90 deg. 0 deg is the equator, +- 90 deg are the
+      // poles. In the map context it goes from  0 to _this.window.height * -2
+      // Equivalent to the Y axis
+      //
+
+      // Turn strings into numbers
+      var longitude = Number(WP.clientGeolocation.longitude);
+      var latitude = Number(WP.clientGeolocation.latitude);
+
+      // Transport longitude to only positive values so it goes from 0 to 360
+      var translatedLongitude = longitude + 180;
+
+      // Transport latitude to only negative values so it goes from -180 to 0
+      var translatedLatitude = latitude - 90;
+
+      // Magic math, jk. It'ssic math.
+      var newX = (translatedLongitude * (_this.window.width * -2)) / 360;
+      var newY = (translatedLatitude * (_this.window.height * 2)) / 180;
+
+      // Move the map
+      _this.moveMap(newX,newY);
```

Comment:
@varffs Yeah, this is exactly what i am talking about in https://github.com/interglobalvision/principled-design-com/issues/17

This error you are talking about is not happening to me tho. Maybe is a problem with your built files? How about you @mikeyrayvon ?